projects/angular/src/data/datagrid/datagrid-page-size.spec.ts

```diff
@@ -56,6 +62,15 @@ export default function (): void {
           expect(context.clarityElement.textContent.trim()).toMatch('Hello world');
         });
 
+        it('displays a select with pageSizeOptionsId as id', function () {
+          const pageSizeOptionsId = 'some-id';
```

Comment:
this uses an autogenerated id by default, right? should we add a test for that?

Reply:
I added a test for the default id.